streams/sidecar_file.proto

```diff
@@ -53,15 +55,43 @@ message TransactionSidecarRecord {
    */
   Timestamp consensus_timestamp = 1;
 
+  SidecarGenerationType generation_type = 2;
+
   /*
   * List of sidecar types.
   * In future there will be other categories.
   */
   oneof sidecar_records {
-    ContractStateChanges state_changes = 2;
+    ContractStateChanges state_changes = 3;
+    ContractActions actions = 4;
+    ContractBytecode bytecode = 5;
```

Comment:
Q: Why do we not have a `ContractBytecodes` wrapper of `repeated ContractBytecode` as we have for `ContractStateChange` and `ContractAction`? How will we externalize all contract bytecodes when a `ContractCall/Create` creates > 1 contracts?

Reply:
Yes, each contract deployment creates a separate record, e.g.
- deploying 5 contracts with a single `ContractCreate` grpc transaction would create the top-level creation record and 4 child creation records; each one will have a corresponding `ContractBytecode` sidecar record.
- deploying 5 contracts via a `ContractCall` grpc transaction would create the top-level call record and 5 creation records; each child will have a corresponding `ContractBytecode` sidecar record.

Reply:
Got it, thanks for the clarifications!